styles/components/forms/slider/slider.json

```diff
@@ -1,9 +1,5 @@
 {
-  "elements": [
-    {
-      "name": "demo",
-      "min": 0,
-      "max": 10
-    }
-  ]
+  "min": 0,
+  "max": 10,
+  "input": "<input type=\"range\" min=\"0\" max=\"10\" value=\"5\">"
```

Comment:
Front-end details in Finnish.

En nyt tarkkaan muista ilman että kaivelis vanhoja tietoja.

Muistaakseni kuitenkin idea on se, että slider-komponentille annetaan form-elementti, ja slider-komponentti muuttaa sen sitten tollaseks jquery ui slideriksi. Kun slideria käyttää niin sen form-elementin arvo muuttuu. Form-elementti voi olla `type="range"` tai type="text" tai vaikka `<select>`.